src/background/hypothesis-chrome-extension.js

```diff
@@ -57,6 +54,9 @@ export default function HypothesisChromeExtension({
     isAllowedFileSchemeAccess,
   });
 
+  /** @type{Map<number, string>} */
+  const isLoading = new Map(); // keeps tracks of what URL each tab is loading
```

Comment:
The variable name `isLoading` suggests that the values are booleans rather than strings. Can I suggest something like `currentlyLoadingUrl`?

Reply:
Done